static/app/components/tours/tour.stories.tsx

```diff
@@ -254,6 +254,60 @@ export const MY_TOUR_KEY = 'tour.my_tour';
       </TourProvider>
     </Fragment>
   ));
+
+  story('Multiple highlighted elements', () => (
+    <Fragment>
+      <p>
+        Most of the time you'll want to highlight a single element. But if you need to
+        highlight multiple elements, you can do so by using the same step ID for each
+        element. Any that you do not want a tooltip for should pass null for the
+        title/description.
+      </p>
+      <TourProvider>
+        <TourElement<MyTour>
+          id={MyTour.NAME}
+          title={null}
+          description={null}
```

Comment:
Totally works, but would you want to just have these be optional? Or is it better to be explicit about omitting tooltip. Up to you but I think I'd be fine if they were not declared altogether

Reply:
I went back and forth on it. I ended up on making it explicit since this should be a niche use case